integration/client/restart_monitor_test.go

```diff
@@ -156,13 +156,21 @@ version = 2
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer container.Delete(ctx, WithSnapshotCleanup)
+	defer func() {
+		if err := container.Delete(ctx, WithSnapshotCleanup); err != nil {
+			t.Logf("failed to delete containerd: %v", err)
```

Comment:
```suggestion
			t.Logf("failed to delete container: %v", err)
```
Might give the wrong impression if this log ever popped up in the CI haha

Reply:
I like it :) Sounds like a real fatal error to me 😹

Reply:
I am incapable of typing the word "container" anymore